Source/WebKit/UIProcess/API/Cocoa/WKContentRuleListStore.mm

```diff
@@ -176,6 +176,13 @@ - (void)_corruptContentRuleListForIdentifier:(NSString *)identifier usingCurrent
 #endif
 }
 
+- (void)_invalidContentRuleListForIdentifier:(NSString *)identifier
```

Comment:
invalidate? createInvalid?
Also, I'd add ForTesting to these selectors so it's clear what they're for.